orchestra/communication/slack.py

```diff
@@ -19,6 +23,13 @@ def __init__(self, api_key=None):
         for attr_name in ('chat', 'groups', 'users'):
             setattr(self, attr_name, getattr(self._service, attr_name))
 
+    def post_message(self, slack_user_id, message, parse='none'):
+        if settings.PRODUCTION:
+            self.chat.post_message(
+                slack_user_id, message, parse='none')
```

Comment:
don't we need something similar in the the mail handler? like if we're on staging we still shouldn't email them, just the console logger